Update changelog (#9878)
learningequality/kolibri: CHANGELOG.md

List of the most important changes for each release.

## 0.15.9
### Added
* Specified pre-commit hook python version to 3.10
* Added Python3.11 to supported python versions
### Fixed
* Fixed PDF completion issues
* Fixed learner-facing metadata display of content duration

> * Fixed "Mark as complete" modal

Reviewer: ```suggestion
* Fixed "Mark as complete" functionality to allow learners to mark resources as complete when allowed by the resource
```

* Disable forward/back buttons on EPUB renderer until locations are properly loaded
* Fix issue that causes learners to skip every other question in an exercise
* Fix searchbox outline
* Fix title spacing in app bar

> * Fix bookmark failure to load data

Reviewer: ```suggestion
* Fix bookmark data loading issues that caused inaccurate bookmark display
```

### Changed

> * Changed _init_.py  from 5 tuple to 3

Reviewer: ```suggestion
* Changed \_\_init\_\_.py  from 5 tuple to 3
```

* Set a max width on Library main content grid to display properly on extra large monitors
* Remove "All options" from filters in Learn search/filtering side panel

> * Switch display of the completion modal to require both completed progress and the 'finished' event

Reviewer: ```suggestion
* Switch display of the completion modal to require both completed progress and the resource to be finished
```

* Add tests to assert totalattempts behaviour

> * Prevent excessive completion modals

Reviewer: ```suggestion
* Display completion modals only on first completion, and allow user to reopen the modal if needed
```
